e2e-polybft/e2e/consensus_test.go

```diff
@@ -28,73 +28,67 @@ import (
 
 var uint256ABIType = abi.MustNewType("tuple(uint256)")
 
-func TestE2E_Consensus_Basic_WithNonValidators(t *testing.T) {
+func TestE2E_Consensus_Basic(t *testing.T) {
 	const (
 		epochSize     = 4
 		validatorsNum = 5
 	)
 
-	cluster := framework.NewTestCluster(t, validatorsNum,
-		framework.WithEpochSize(epochSize),
-		framework.WithNonValidators(2),
-		framework.WithTestRewardToken(),
+	var (
+		initMinterBalance = ethgo.Ether(4e15)
+		premineBalance    = ethgo.Ether(2e15)
```

Comment:
This is quite a lot, bear in mind that `ethgo.Ether` converts provided value expressed in the tokens amount to weis (that is, it already mutiplies provided value with 10^18).

```suggestion
		initMinterBalance = ethgo.Ether(4)
		premineBalance    = ethgo.Ether(2)
```

You can change this when fixing e2e tests.